src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/locations/LocationsApiService.kt

```diff
@@ -5,21 +5,22 @@ import org.springframework.beans.factory.annotation.Qualifier
 import org.springframework.data.domain.PageImpl
 import org.springframework.stereotype.Service
 import org.springframework.web.reactive.function.client.WebClient
+import uk.gov.justice.digital.hmpps.prisonertonomisupdate.locations.model.LegacyLocation
 import uk.gov.justice.digital.hmpps.prisonertonomisupdate.locations.model.Location
 import uk.gov.justice.digital.hmpps.prisonertonomisupdate.services.RestResponsePage
 import uk.gov.justice.digital.hmpps.prisonertonomisupdate.services.typeReference
 
 @Service
 class LocationsApiService(@Qualifier("locationsApiWebClient") private val webClient: WebClient) {
-  suspend fun getLocation(id: String, includeHistory: Boolean = false): Location {
+  suspend fun getLocation(id: String, includeHistory: Boolean = false): LegacyLocation {
     return webClient.get()
       .uri {
-        it.path("/locations/{id}")
+        it.path("/sync/id/{id}")
```

Comment:
There is a new locations endpoint which returns the location in a more 'nomis friendly' format so switching to use that